File: El Mamoun/Lv 1/Problem 12.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

/*
 Problem 12: Write a program to ask the user to enter:• Number1 , Number2 Then Print the Max Number
*/
namespace Problem_12
{
    internal class Program
    {
        static int ReadNumber()
        {
            int Num;
            Console.Write("Please Enter a Number? ");
            Num = Convert.ToInt32(Console.ReadLine());

            return Num;
        }
        static int Max2Num(int Num1,int Num2)
        {
            return (Num1 > Num2) ? Num1 : Num2;
        }
        static void PrintResult(int Max)
        {
            Console.Write(Max);  
        }
        static void Main(string[] args)
        {
            PrintResult(Max2Num(ReadNumber(), ReadNumber()));

            Console.ReadKey();
        }
    }
}

File: El Mamoun/Lv 1/Problem 12.cpp
#include<iostream>
using namespace std;

/*
 Problem 12: Write a program to ask the user to enter:• Number1 , Number2 Then Print the Max Number
*/

int ReadNumber()
{
	int number = 0;

	cout << "Please Enter a Number? ";
	cin >> number;

	return number;

}

int Max2Number(int Num1,int Num2)
{

	if (Num1 > Num2)
		return Num1;

	else
		return Num2;

}

void PrintMaxNumber(int Max)
{
	cout << "\nMax Number is = " << Max << endl;
}

int main()
{

	int Num1, Num2;

	Num1 = ReadNumber();
	Num2 = ReadNumber();

	PrintMaxNumber(Max2Number(Num1, Num2));

	return 0;
}